janusgraph-inmemory/src/main/java/org/janusgraph/diskstorage/inmemory/InMemoryKeyColumnValueStore.java

```diff
@@ -104,17 +135,225 @@ public void close() throws BackendException {
         kcv.clear();
     }
 
+    public InMemoryKeyColumnValueStoreFragmentationReport createFragmentationReport(StoreTransaction txh) throws BackendException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'createFragmentationReport(StoreTransaction)' has an NPath complexity of 202](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=3231392)